### ISSUE TITLE
create blank page option for new tab page

### ISSUE THREAD

**Issue by rebron** [brave/brave-browser#4509]
## Description
Provide users with the option to set blank page as the New Tab Page.

## Designs
[TBA]

Abstract link for more CSS/layout: https://share.goabstract.com/9563ae1d-93af-40ef-811b-4c3f384fe8f4

**Comment by rebron** [comment]
related issues: https://github.com/brave/brave-browser/issues/353 and https://github.com/brave/brave-browser/issues/1872

**Comment by bcrownover** [comment]
I was doing quite a bit of work to get Chrome to do what Brave does by default. The one thing that really disappointed me about Brave was that I want a blank new tab page. I've got one now via extension, but I'm looking forward to this just being an option.

**Comment by bsclifton** [comment]
Related https://github.com/brave/brave-browser/issues/2999 - you could pick `Blank page`

**Comment by washere** [comment]
Disabling images then it gets worse. Why does it take eons to set a solid background color (most would want black) by the devs?

You do realize NUMEROUS users, probably millions, DROP Brave because the NTP is a barbaric reddish_purple which is blinding. Over 90% of professional coders, IT staff, pro writers, etc. etc. many of whom are major social influencers ONLY work with dark themed: IDEs, Productivity software, platforms etc. So not only Brave does not go viral by them, it is effectively sabotaged internally from spreading, due to a tiny lacking feature. Also needs a theme option with search engine text_field on a NTP, but that's a luxury right now, when we can not change the red/purple torch in our eyes on a NTP.

**Comment by rebron** [comment]
@washere follow-up issue specifically addressing background color here -> https://github.com/brave/brave-browser/issues/8061 
We have a/some designs that we'll be posting in that issue shortly.

**Comment by washere** [comment]
@rebron Thanks for the reply.

*** NO "dark" designs**, as you put it, is not what many online (github, reddit, forums etc etc) want.
*** Nor "dark photos"** etc. as many online have said.

Many of us just want to be able to:

**SET THE NTP BACKGROUND TO BLACK**

* Other colors would be a luxury right now. Just like Windows Background option to set to a solid color. 

WE JUST WANT TO HAVE BLACK BACKGROUND & BE ABLE TO REMOVE OTHER "DESIGN WIDGETS" TOO. Simple, should take a few minutes to code.

WE DO NOT WANT DARK DESIGNS.
JUST PLAIN BLACK SOLID COLOR COLOR & NO BELLS OR WHISTLES WIDGETS if we choose to remove them. Like a minimal Morpheon Theme design:

![image](https://user-images.githubusercontent.com/5448047/73799086-cac3d480-47ac-11ea-8f5e-ed601d49505b.png)

Next step would be the default Search_engine Text_Field:

![image](https://user-images.githubusercontent.com/5448047/73799112-e7f8a300-47ac-11ea-9fbd-28ec615e8ccc.png)

THANK YOU VERY MUCH for you taking up this CRITICAL issue for many influential users online.
You are much appreciated.

**Comment by rebron** [comment]
@washere We'll take your feedback and yours is one of many and we appreciate it. But writing in all caps doesn't work for me.

Also, you mentioned `NO "dark" designs**, as you put it` yet you put a screenshot of minimal Morpheon Theme design. There's surprisingly a lot of shades of black or dark which is why we need designs and it's why you posted one.

Many years ago we built extensions for this exact purpose, giving users what they want. I know there are extensions available now that can give you the dark/black new tab page that you want. You and others don't need to wait on #8061 to get what you want today.

**Comment by washere** [comment]
@rebron 

Thanks for your reply. All caps not working for you does not matter to me as if a company does not want to give the basic:

* SET BACKGROUND TO SOLID COLOR

Which (black or very dark themes) is a standard for over 90% of coders setups not to mention professional writers etc, then I assume it will not be done for A VERY LONG TIME regardless.

* I did not say recreate Morpheon, it was just a minimalist example.

* Further, because there are MANY SHADES OF BLACKISH?DARK is why I suggested initially have pure 
 black #000000 as an option.

* Then other solid colors (palette menu options as in Win10 solid color options) in the releases after that, just to make things simple.

So a simple request:

* BLACK (#000) BACKGROUND

Becomes in your own word: "why we need designs", despite repeating saying give us minimal pure black background, NOT DESIGNS. I knew despite repeating it would get back to "DESIGNS".

I did not expect it to be accepted, not twisted never mind taken up.
As a long time coder and designer, it seems designers are creating job-bags for themselves and the project managers are not in charge.

Further BASIC options:
* OPTION TO REMOVE ALL WIDGETS in the new All-Black NTP
+
* LATER PHASE: Default search engine text_field
were not expected by to me to be taken up either.

I would like to say millions of pro writers, IT staff, coders etc who only with black/dark theme tools will not use Brave for this simple reason. As I do not for my daily driver.

Good luck with "the designs" and hope spelling it out makes clear what I requested and you misrepresented as what you have scheduled are two different things and that does it for you.

Best wishes.

**Comment by washere** [comment]
I repeat my request: No designs please, just minimal page, with pure black (#000000) solid color background option please. Thanks.

**Comment by madhums** [comment]
Looking at the label "needs design" and title of the issue "blank page" do we really need a design? I'd say there should be an option to simply use the OS color scheme (dark or light) OR perhaps a hex input to provide a color for page bg.

**Comment by karenkliu** [comment]
is this a dupe of https://github.com/brave/brave-browser/issues/2999 ?

**Comment by bsclifton** [comment]
@karenkliu yup- duplicate
